lib/elixir/lib/option_parser.ex

```diff
@@ -71,6 +71,9 @@ defmodule OptionParser do
   Note that you should only supply the `:switches` or the`:strict` option.
   If you supply both, an `ArgumentError` exception will be raised.
 
+    iex> OptionParser.parse(["--source", "lib"], switches: [source: :string], strict: [source: :string])
+    ** (ArgumentError) :switches and :strict cannot be given together
+
```

Comment:
Right, but in this specific case it feels like it doesn't add to me. But let's have others decide, in the meantime, the indentation is wrong (should be 2 more spaces) :).